modules/likes/jetpack-likes-settings.php

```diff
@@ -0,0 +1,645 @@
+<?php
+
+class Jetpack_Likes_Settings {
+	function __construct() {
+		$this->in_jetpack = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? false : true;
```

Comment:
The code in this file is mostly moved from `likes.php`, and I opted not to change it in order to avoid introducing more testing steps for this PR (or bugs in something that was already functioning properly :smile:). Anyway, I've refactored it to use the first suggestion in 9aaef66b8485d7aa1eab7ec9eba6d4648ae72d24. I avoided the second one since it will generate notices in environments in which `IS_WPCOM` constant is not defined.